### PR TITLE
textarea auto height implemented

### PR DESCRIPTION
**Pull request by evidentti**

### CODE REVIEW

File: app/scripts/textarea.directive.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+angular.module('dashboard')
+    .directive('dbTextarea', ['$timeout', function ($timeout) {
+        return {
+            scope: false,
+            restrict: 'A',
+            replace: 'true',
+            link: function (scope, element/*, attrs*/) {
+                scope.initialHeight = scope.initialHeight || element[0].style.height;
+                var resize = function resize() {
+                    element[0].style.height = scope.initialHeight;
```
Comment (tkenakka) [inline app/scripts/textarea.directive.js]: Property re-set on following line so isn't this useless?

**Comment by tkenakka** [comment]
LGTM but as discussed "input change" event might need to be replaced using model watching

**Comment by evidentti** [comment]
Remark implementation added. Improvements.

**Comment by tkenakka** [comment]
LGTM

**Pull request merged**